openshift/jenkins/pims-app/Jenkinsfile

```diff
@@ -0,0 +1,173 @@
+// Implementation Details
```

Comment:
Does this replace frontend/jenkinsfile? if so feel free to remove that file.

Reply:
I didn't want to remove your files until this was working after the merge.